src/Categories/Category/Monoidal/Star-Autonomous.agda

```diff
@@ -0,0 +1,35 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category
+open import Categories.Category.Monoidal.Core using (Monoidal)
+open import Categories.Category.Monoidal.Symmetric
+
+module Categories.Category.Monoidal.Star-Autonomous {o ℓ e} {C : Category o ℓ e} (M : Monoidal C) where
+
+open import Level
+
+open import Categories.Category.Product
+open import Categories.Functor
+open import Categories.Functor.Properties
+open import Categories.NaturalTransformation.NaturalIsomorphism
+open import Categories.Morphism C
+open import Categories.Functor.Hom
+open Hom C
+
+open Category C renaming (op to Cᵒᵖ) hiding (id)
+open Monoidal M
+open Functor ⊗ renaming (op to ⊗ₒₚ)
+
+open import Relation.Binary.Structures
+
+record Star-Autonomous : Set (levelOfTerm M) where
+  field
+    symmetric : Symmetric M
```

Comment:
I would think that you'd want this `Symmetric` to be part of the module parameters too? Are there cases where a single monoidal category has different `Symmetric` witnesses that lead to different Star-Autonomous structures?

Reply:
I'm not sure, I just opted to do things this way because after looking a bit for inspiration I found that Compact Closed Categories did have a field called `symmetric`. I'll gladly take your suggestion if you think that's the best approach :smile:

Reply:
Also would that change imply I move the module to under `Monoidal.Symmetric` ?

Reply:
"best" is definitely open for debate! But for Star-Autonomous always assume SMCC, then yes to both.

Reply:
Yes, I think so.